src/number_theory/legendre_symbol/jacobi_symbol.lean

```diff
@@ -0,0 +1,367 @@
+/-
+Copyright (c) 2022 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import number_theory.legendre_symbol.quadratic_reciprocity
+import data.zmod.coprime
+
+/-!
+# The Jacobi Symbol
+
+We define the Jacobi symbol and prove its main properties.
+
+## Main definitions
+
+We define the Jacobi symbol, `jacobi_sym a b` for integers `a` and natural numbers `b`
+as the product over the prime factors `p` of `b` of the Legendre symbols `zmod.legendre_sym p a`.
+This agrees with the mathematical definition when `b` is odd.
+
+The prime factors are obtained via `nat.factors`. Since `nat.factors 0 = []`,
+this implies in particular that `jacobi_sym a 0 = 1` for all `a`.
+
+## Main statements
+
+We prove the main properties of the Legendre symbol, including the following.
```

Comment:
```suggestion
We prove the main properties of the Jacobi symbol, including the following.
```

Reply:
Thanks! I don't know why I wrote "Legendre"; I should have re-read that carefully (but you see what you expect...).